src/machines/authMachine.ts

```diff
@@ -235,3 +231,12 @@ async function getAndSyncStoredToken(input: {
   localStorage.setItem(TOKEN_PERSIST_KEY, fileToken)
   return fileToken
 }
+
+async function logout() {
+  localStorage.removeItem(TOKEN_PERSIST_KEY)
+  if (isDesktop()) return Promise.resolve(null)
```

Comment:
User's can't logout on desktop? :thinking:

Reply:
lol this function was never invoked, therefore it was impossible to log out on web. 🤦🏻. I've fixed it now, by making the `loggingOut` intermediate state which invokes either this or the desktop logout behavior. I'm not sure why we have decided to not hit the `/logout` API endpoint on desktop and instead only delete the token locally, but I don't want to mess with the current behavior in this PR.